internal/flypg/node.go

```diff
@@ -350,10 +370,32 @@ func (n *Node) PostInit(ctx context.Context) error {
 				return fmt.Errorf("failed to issue registration certificate: %s", err)
 			}
 		} else {
-			// Configure as standby
-			log.Println("Registering standby")
-			if err := n.RepMgr.registerStandby(); err != nil {
-				return fmt.Errorf("failed to register new standby: %s", err)
+			if n.Witness {
```

Comment:
Out of curiosity have you tested switching a nodes role between member and witness? Does that break things?

Reply:
It would break that node at the very least.